src/js/themes/base.js

```diff
@@ -274,7 +276,12 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
       },
       input: {
-        padding: `${baseSpacing / 2}px`,
+        padding: {
```

Comment:
I believe we need to adjust it on base.d.ts.

Reply:
Yes, you're right. Let me update.